### ISSUE TITLE
bug(yt): share button opens up the speed selection menu in some cases

### ISSUE THREAD

**Issue by SodaWithoutSparkles** [ReVanced/revanced-patches#2824]
### Type

Error at runtime

### Bug description

The share button next to the dislikes brings up the video playback speed panel, if you opened the speed selection panel before

### Steps to reproduce

1. Click player settings
2. Enter video playback speed submenu
3. Exit the submenu
4. Click the share button
5. Video playback speed menu was shown instead of the expected share menu.

### Relevant log output

```shell
N/A
```

### Screenshots or videos

https://github.com/ReVanced/revanced-patches/assets/64138578/8ff3ffcc-6074-404c-9518-01534bed7195

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by xDARKxDEVILx** [comment]
Easily reproduced for me

**Comment by Domiiniik** [comment]
I can reproduce the issue but it only happens if: I enter video Speed submenu first, then decide to back out of it, and then proceed to click on share button, which results in showing the Speed menu but only once, if i back out and click on share menu again the issue No longer occurs until i repeat previously mentioned steps to reproduce.

**Comment by SodaWithoutSparkles** [comment]
Yes Domiiniik, thanks for the detailed description. The video you shared shows the exact same issue as i did.

**Comment by johnconner122** [comment]
It doesn't happen to me on latest version. Can you that it still happens?
